v7/base-austere/templates/index_helper.tmpl

```diff
@@ -0,0 +1,19 @@
+## -*- coding: utf-8 -*-
```

Comment:
I would keep that mathjax_script helper and not change this template. Perhaps some people want to switch from `base` to `base-austere`, and they will get unusual errors about missing things.